docs/source/daml/intro/2_Scenario.rst

```diff
@@ -110,13 +114,13 @@ To test for failing submits and keep the scenario running thereafter, or fail if
 Archiving contracts
 -------------------
 
-Archiving contracts works just like creating them, but using ``archive`` instead of ``create``. Where ``create`` takes an instance of a template, ``archive`` takes a reference to a contract.
+Archiving contracts works just like creating them, but using ``archiveCmd`` instead of ``createCmd``. Where ``createCmd`` takes an instance of a template, ``archiveCmd`` takes a reference to a contract.
```

Comment:
Since when is `archiveCmd` a thing?

Reply:
Since this morning :slightly_smiling_face: Documentation-driven development. I got annoyed by 3 things during the migration that I added since:
1. `archiveCmd`, very reasonable to have imho.
2. The absence of an equivalent of `scenario`, we now have `script`.
3. No equivalent of `pass`, we now have `passTime`.